storage/src/cache/worker.rs

```diff
@@ -84,12 +83,12 @@ impl AsyncPrefetchMessage {
     }
 }
 
+/// An asynchronous tasks pool manager which is only for prefetch IO right now.
```

Comment:
An asynchronous task manager for data prefetching?

Reply:
SGTM